packages/tupaia-web/src/components/Modal.tsx

```diff
@@ -34,12 +37,13 @@ const CloseButton = styled(IconButton)`
   min-width: initial;
   position: absolute;
   top: 0.1rem;
-  right: 0.1rem;
+  right: 0rem;
```

Comment:
Any reason to move the close button? I think that i's closer to the design with the 0.1rem to the left isn't it?

Reply:
Yeah for whatever reason, it didn't look the way I wanted it to but I think its because of the removal of scrollbar that made it look janky. But I've changed it back to the original position.